src/core/graph.test.js

```diff
@@ -316,7 +316,93 @@ describe("graph", () => {
       });
     });
 
-    describe("in- and out-edges", () => {
+    describe("getInEdges and getOutEdges", () => {
+      describe("type filtering", () => {
+        class ExampleGraphConstructor {
```

Comment:
This name is confusing to me. Isn't it more like an `ExampleGraph`? I expected to see `new ExampleGraphConstructor().createGraph()` or something.